### PR TITLE
Fixed API dependencies

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

**Comment by core23** [comment]
As the REST bundle is not required for the this bundle, the api form and controllers definitions should only be loaded, when the bundles are used.

File: DependencyInjection/SonataClassificationExtension.php [hunk 1]
```diff
@@ -47,8 +47,11 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('orm.xml');
         $loader->load('form.xml');
         $loader->load('serializer.xml');
-        $loader->load('api_controllers.xml');
-        $loader->load('api_form.xml');
+        
+        if (isset($bundles['FOSRestBundle']) && isset($bundles['NelmioApiDocBundle'])) {
```
Comment (Soullivaneuh) [inline DependencyInjection/SonataClassificationExtension.php]: Not sure that `NelmioApiDocBundle` is required to get REST working.

**Comment by core23** [comment]
@Soullivaneuh Any better idea? It is the same logic like in the news bundle...

**Comment by core23** [comment]
ping @Soullivaneuh

**Comment by Soullivaneuh** [comment]
I would like to hear @rande opinion for this.

**Comment by core23** [comment]
ping @rande

**Comment by Soullivaneuh** [comment]
@OskarStark please avoid to randomly assign contributors just if they answer on a PR.

Assign only if issues / PR is closely related to a specific contributor. And if it's not sure, comment an explanation of why.

BTW in this case, I would assign @rande for last review.

**Pull request merged**